### PR TITLE
1682 feedback python

### LINKED ISSUE #1726: Extension overrides settings.json when adding App Service
When working on #1618 we realised that now that Flask has a settings.json, the extension overrides it when creating the one for App Service.

### PR DESCRIPTION
**Pull request by dgomezc**
## PR checklist

- Quick summary of changes
  - Refactor flask templates
- Which issue does this PR relate to?
#1618  - Feedback on generated Python code

### CODE REVIEW

File: templates/Web/_composition/Flask/Page.Flask.List.AddRoutes/backend/server_postaction.py [hunk 1]
```diff
@@ -1,43 +1,41 @@
...
+
+@app.route(constants.ENDPOINT_LIST + "/<string:item_id>", methods=["DELETE"])
+def delete_list_item(item_id):
+    item_to_remove = next(
+        (item for item in sample_data.sample_list if item["id"] == item_id),
+        None,
+    )
+    if item_to_remove is None:
+        json_response = flask.jsonify(
+            {"error": "Could not find an item with the given id"}
```
Comment (prupipho) [inline templates/Web/_composition/Flask/Page.Flask.List.AddRoutes/backend/server_postaction.py:35]: maybe this could be a constants too?

**Review by prupipho — COMMENTED** [review]
All looking good so far. Gonna test it now.

**Review by prupipho — DISMISSED** [review]
Tested and reviewed

**Review by prupipho — DISMISSED** [review]

**Review by sibille — CHANGES_REQUESTED** [review]
After adding the server to src, the app does not start when published to azure app service

**Review by sibille — APPROVED** [review]

**Pull request merged**